test_scripts/Polices/External_UCS/009_ATF_P_TC_SnapshotPT_without_external_consent_status_groups_struct.lua

```diff
@@ -0,0 +1,94 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [Policies] External UCS: SnapshotPT without "external_consent_status_groups" param
```

Comment:
@dboltovskyi, please add reference to requirement "[Policies] External UCS: PreloadedPT without "external_consent_status_groups" struct" because you are using as precondition of this script, point 1.